crates/prql-compiler/tests/integration/docker-compose.yml

```diff
@@ -61,3 +61,15 @@ services:
       # ClickHouse can load csv only from user_files_path (default `/var/lib/clickhouse/user_files/`)
       # https://clickhouse.com/docs/en/operations/server-configuration-parameters/settings#server_configuration_parameters-user_scripts_path
       - ./data/chinook:/var/lib/clickhouse/user_files/chinook/:ro
+  glaredb:
+    build:
+      context: dockerfiles
+      dockerfile: glaredb.Dockerfile
+    ports:
+      - "6543:6543"
```

Comment:
I changed this to:

```
  glaredb:
    build:
      context: dockerfiles
      dockerfile: glaredb.Dockerfile
    ports:
      - "6543:6543"
    volumes:
      - ./data/chinook:/data/chinook
```

(adds a volume)

And the nginx shouldn't be needed since glaredb can just read from the volume directly.